recipes/twitch-libsoundtrackutil/all/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+include(conanbuildinfo.cmake)
+
+conan_basic_setup(TARGETS)
+add_subdirectory("source_subfolder")
```

Comment:
The cmake script of twitch runs conan itself: see https://github.com/twitchtv/libsoundtrackutil/blob/a035f2e01417768fc82e515e44d720ffa69f7c92/cmakeUtils/setup.cmake#L22
This is unwanted as the conan requirement in the cmake script might/will have different options as the one of this conan cci recipe.
So please patch out running of conan inside the cmake script.

Reply:
Inside the function we check `CONAN_EXPORTED` and skip running conan in that case.